tests/testthat/test-calculate_stay.R

```diff
@@ -117,6 +127,24 @@ test_that("Can calculate the correct stay for vectors of dates", {
     ),
     c(62, 62, 61, 62)
   )
+
+  # SC calculation - if sc_qtr is supplied but end_date is not missing use end_date
+  # Start before FY, end during FY
+  expect_equal(
+    calculate_stay(
+      "1920",
+      as.Date(c(
+        "2019/03/31", "2019/06/30",
+        "2019/01/01", "2019/04/01"
+      )),
+      as.Date(c(
+        "2019/10/31", "2019/08/31",
+        "2020/04/01", "2020/07/01"
+      )),
+      c("2019Q1", "2019Q2", "2019Q3", "2019Q4")
+    ),
+    c(214, 62, 456, 457)
+  )
 })
 
 test_that("calculate stay function works", {
```

Comment:
Could you replace the tibble here with `tribble` please, as it's much easier to read (and add to):
```R
test_tibble <- tibble::tribble(
  ~start_date, ~end_date,  ~sc_latest_submission, 
  2019-03-31,  2019-10-31, NA,                 
  2019-06-30,  2019-08-31, NA,                 
  2019-01-01,  2020-04-01, NA,                 
  2019-04-01,  2020-07-01, NA,                 
  2019-03-31,  NA,         NA,                 
  2019-06-30,  NA,         NA,                 
  2019-01-01,  NA,         NA,                 
  2019-04-01,  NA,         NA,                 
  2019-03-31,  NA,         "2019Q1",             
  2019-06-30,  NA,         "2019Q2",             
  2019-01-01,  NA,         "2019Q3",             
  2019-04-01,  NA,         "2019Q4",             
  2019-07-31,  NA,         "2019Q1",             
  2019-10-31,  NA,         "2019Q2",             
  2020-01-31,  NA,         "2019Q3",             
  2020-04-30,  NA,         "2019Q4",             
  2019-03-31,  2019-10-31, "2019Q1",             
  2019-06-30,  2019-08-31, "2019Q2",             
  2019-01-01,  2020-04-01, "2019Q3",             
  2019-04-01,  2020-07-01, "2019Q4"
)
```

Reply:
I've split out the tribble for normal/sc calculation. Could you please check this again? I can't get it to work properly and the tests keep failing. However, i have fixed the other tests this was failing on here too